css/80_app.css

```diff
@@ -5777,6 +5777,12 @@ li.hide + li.version .badge .tooltip .popover-arrow {
 body {
     scrollbar-width: 10px;
```

Comment:
Yeah, I can’t find anything that suggests this was ever a valid or supported syntax in CSS. Feel free to remove it.